### DISCUSSION TITLE
开放函数无法使用

### DISCUSSION THREAD

**Discussion by daweilang** [TaleLin/lin-ui#1348] [求助] [answered]
使用文档的方案测试  TypeError: toast.linShow is not a function ，
已经升级到0.9.4版本
json和页面也都引用组件
<l-toast id="my-toast" />

**Comment by juzi214032** [comment, marked as answer]
麻烦贴一下代码

**Reply by Black-Dong** [reply]
您好，请问这个问题您是怎么解决的呢，我也遇到了相同的问题
`
const toast = that.selectComponent('#evaluateToast');
          toast.linShow({
            title: '请重新尝试'
          })
`

**Reply by daweilang** [reply]
应该是升级以后缓存问题，或者是升级不完全npm重构有问题，重新更新以后没有发生这个问题。忘了在这回复。
